src/java/com/unifina/signalpath/blockchain/Web3jHelper.java

```diff
@@ -323,6 +324,29 @@ public static Type web3jArrayGet(Array array,int... indices){
 		return (Type) val;
 	}
 
+	public static Web3j getWeb3jConnectionFromConfig(){
+		EthereumModuleOptions ethereumOptions = new EthereumModuleOptions();
+		return Web3j.build(new HttpService(ethereumOptions.getRpcUrl()));
+	}
+
+	/**
+	 * calls operator() getter from contract with public "operator" address:
+	 * address public operator;
+	 *
+	 * @return
+	 */
+	public static String getOperatorAddress(Web3j web3j, String contractAddress) throws IOException {
+		Function getOperator = new Function("operator", Arrays.<Type>asList(), Arrays.<TypeReference<?>>asList(TypeReference.create(Address.class)));
```

Comment:
This is specific to the monoplasma contract, so this method should not be in this helper, but rather live in a service specific to the use case. Maybe the right place is `CommunityJoinRequestService` for now, although it can be moved to a more general service like `CommunityService` if it turns out that it's needed from multiple different places.